source/documentation/deploying_apps/index.md

```diff
@@ -68,6 +74,103 @@ A possible exception to this is if your org is mature and has pre-existing space
+
+### Specify a private route in the private app's manifest
+
+A [route](https://docs.cloudfoundry.org/devguide/deploy-apps/routes-domains.html#routes) [external link] is an address associated with a Cloud Foundry app. Cloud Foundry uses routes to send requests to apps.
+
+You must specify a private route in the private app's manifest to tell Cloud Foundry that this app should not be accessible from the internet.
+
+Create the private app's manifest with the following code:
+
+```
+---
+applications:
+- name: PRIVATE_APPNAME
+  routes:
+  - route: PRIVATE_APPNAME.apps.internal
+```
+
+If the private app was previously a public app, you must also run the following to remove the pre-existing public route:
+
+```
+cf unmap-route PRIVATE_APPNAME DOMAIN --hostname HOSTNAME
+```
+
+### Set the private app URL in the public app's manifest
+
+The public app must read the private app URL from an environment variable in the public app's manifest.
+
+<br>
+
+<style>
+.govuk-warning-text{font-family:nta,Arial,sans-serif;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale;font-weight:400;font-size:16px;font-size:1rem;line-height:1.25;color:#0b0c0c;position:relative;margin-bottom:20px;padding:10px 0}@media print{.govuk-warning-text{font-family:sans-serif}}@media (min-width:40.0625em){.govuk-warning-text{font-size:19px;font-size:1.1875rem;line-height:1.31579}}@media print{.govuk-warning-text{font-size:14pt;line-height:1.15;color:#000}}@media (min-width:40.0625em){.govuk-warning-text{margin-bottom:30px}}.govuk-warning-text__assistive{position:absolute!important;width:1px!important;height:1px!important;margin:-1px!important;padding:0!important;overflow:hidden!important;clip:rect(0 0 0 0)!important;-webkit-clip-path:inset(50%)!important;clip-path:inset(50%)!important;border:0!important;white-space:nowrap!important}.govuk-warning-text__icon{font-family:nta,Arial,sans-serif;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale;font-weight:700;display:inline-block;position:absolute;top:50%;left:0;min-width:32px;min-height:29px;margin-top:-20px;padding-top:3px;border:3px solid #0b0c0c;border-radius:50%;color:#fff;background:#0b0c0c;font-size:1.6em;line-height:29px;text-align:center;-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none}@media print{.govuk-warning-text__icon{font-family:sans-serif}}.govuk-warning-text__text{display:block;margin-left:-15px;padding-left:65px}
+</style>
+
+<div class="govuk-warning-text">
+  <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+  <strong class="govuk-warning-text__text">
+    <span class="govuk-warning-text__assistive">Warning</span>
+    <p>Connections between public and private apps are not encrypted by default.</p>
+    <p>If you need encrypted connections between your apps, it is your responsibility to implement this.</p>
+  </strong>
+</div>
+
+You must specify:
+
+- the private app URL as a non-secure `http` URL
```

Comment:
We should probably call out the special `apps.internal` domain here too maybe. We haven't introduced it anywhere other than the manifest example.

Reply:
would this be a call to action or supporting information? are we telling the user to do anything extra?

Reply:
Yeah, like in your commit - just some supporting information to make it clear that the `apps.internal` domain is a special thing.